setup.py

```diff
@@ -18,23 +18,13 @@
 
 import sys
 from setuptools import setup
+from os import path
 
-DESCRIPTION = "Pandas DataFrame API on Apache Spark"
+DESCRIPTION = "Koalas: pandas API on Apache Spark"
 
-LONG_DESCRIPTION = """
-Koalas makes data scientists more productive when interacting with big data,
-by augmenting Apache Spark's Python DataFrame API to be compatible with
-Pandas'.
-
-Pandas is the de facto standard (single-node) dataframe implementation in
-Python, while Spark is the de facto standard for big data processing.
-With this package, data scientists can:
-
-- Be immediately productive with Spark, with no learning curve, if one
-  is already familiar with Pandas.
-- Have a single codebase that works both with Pandas (tests, smaller datasets)
-  and with Spark (distributed datasets).
-"""
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
```

Comment:
we should do a test publication to pypi and see what it looks like. I worry readme.md is way too long

Reply:
ah, I see. let me upload to TestPyPI.

Reply:
as discussed offline, this looks fine ...